config/constants/src.py

```diff
@@ -0,0 +1,147 @@
+
+    Атрибуты:
+        FILE_LINK (int): Максимальная длина ссылки на файл.
+        MAX_NAME (int): Максимальная длина имени.
+        MAX_NAME_COMPANY (int): Максимальная длина названия компании.
+        MAX_TELEGRAM_USERNAME (int): Максимальная длина Telegram username.
+        MIN_PASSWORD (int): Минимальная длина пароля.
+        SLUG (int): Максимальная длина slug.
+    """
+
+    FILE_LINK: int = 2048
+    MAX_NAME_COMPANY: int = 255
+    MAX_TELEGRAM_USERNAME: int = 100
+    MIN_PASSWORD: int = 8
+    SLUG: int = 110
+
+
+@dataclass(frozen=True)
+class MiscConstantsBase(ConstantsBase.MiscConstants):
+    """Базовый класс разных общесистемных констант, используемых в пакете `src`.
+
+    Класс реализован, как неизменяемый.
+
+    Так же класс наследует значения из ConstantsBase.MiscConstants.
+
+    Атрибуты:
+        BASE_DIR (Path): Корневая директория проекта.
+    """
+
+    BASE_DIR: Path = Path(__file__).resolve().parents[2]
+
+
+class SummaryBase(ConstantsBase.Summary):
+    """Базовый класс констант кратких описаний, используемых в пакете `src`.
+
+    Так же класс наследует значения из ConstantsBase.Summary.
+    """
+
+
+class TextErrorBase(ConstantsBase.TextError):
+    """Базовый класс для хранения стандартных текстов ошибок,
+    используемых в пакете `src`.
+
+    Так же класс наследует значения из ConstantsBase.TextError.
+
+    Атрибуты:
+        EXISTS_EMAIL (str): Сообщение о существующем email.
+        INVALID_PASSWORD (str): Сообщение о невалидном пароле.
+        NOT_FOUND (str): Шаблон сообщения об отсутствии объекта.
+    """
+
+    EXISTS_EMAIL: str = 'Пользователь с такой электронной почтой уже существует.'
+    INVALID_PASSWORD: str = (
+        'Пароль должен содержать символы латинского алфавита в обоих регистрах, '
+        f'числа и иметь минимальную длину в {LengthBase.MIN_PASSWORD} символов.'
+    )
+    NOT_FOUND: str = 'Не найден объект {obj} по данному id: {id}'
+
+
+class TitleBase(ConstantsBase.Title):
+    """Базовый класс констант-заголовков, используемых в пакете `src`.
+
+    Так же класс наследует значения из ConstantsBase.Title.
+    """
+
+
+@dataclass(frozen=True)
+class ValidationBase(ConstantsBase.Validation):
+    """Базовый класс для валидации данных, используемых в пакете `src`.
+
+    Класс реализован, как неизменяемый.
+
+    Так же класс наследует значения из ConstantsBase.Validation.
+
+    Атрибуты:
+        EMAIL_REGEX (str): Регулярное выражение для валидации email.
+        PHONE_NUMBER_PATTERN (str): Регулярное выражение для российских номеров (начинаются с 7).
+        PHONE_REGEX (str): Регулярное выражение для номеров в формате +7.
+        TELEGRAM_USERNAME_PATTERN (str): Регулярное выражение для Telegram username.
+    """
+
+    EMAIL_REGEX = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
+    # Нужно понять, какое выражение оставляем для номера телефона с + или без
+    PHONE_NUMBER_PATTERN = r'^7\d{10}'
+    PHONE_REGEX = r'^\+7\d{10}$'
```

Comment:
Я как то писал регулярку. Может всё же её использовать будем?
```
^(?!.*([\-\(\) ])\1)(?=.*\d)\+*[\d\-\(\) ]+$
```

Reply:
Я не правил эти регулярки, они уже были написаны кем-то.  
Нужно, во-первых, понять, нам надо использовать номер телефона по маске с +7 или просто7?

Reply:
В проекте так же была вторая регулярна телефонного номер r'^7\d{10}' = формат 79123456789 (без +? 10 символов после семерки)

PHONE_NUMBER_PATTERN = r'^7\d{10}' используется в файле - Tabit/src/schemas/validators/admin_company.py
PHONE_REGEX = r'^\+7\d{10}$' используется в файле - Tabit/src/schemas/landing_page.py

Reply:
оставляю в проекте 
```suggestion
    PHONE_REGEX = r'^\+7\d{10}$'
```
(строго 7 и 10 цифр (+71234567890); подходит, в принципе,  для любых стран 70000000000)

Reply:
Оставляй вариант Льва. Не надо сейчас загонять себя в строгие рамки. Допиши TODO откорректировать формат по требованию фронтэнда

Reply:
внес изменения в код

`PHONE_NUMBER_PATTERN: str = r'^(?!.*([\-\(\) ])\1)(?=.*\d)\+*[\d\-\(\) ]+$'`